### PR TITLE
Don't rerun certificate manager tests 1000 times.

### PR DESCRIPTION
**Pull request by rmmh**
**What this PR does / why we need it**:
Running every testcase 1000 times needlessly bloats the logs.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: pkg/kubelet/certificate/certificate_manager_test.go [hunk 1]
```diff
@@ -202,21 +202,22 @@ func TestSetRotationDeadline(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		for i := 0; i < 1000; i++ {
```
Comment (jcbsmpsn) [inline pkg/kubelet/certificate/certificate_manager_test.go]: Move this line down by one line and the test case won't repeat in the logs.

Comment (rmmh) [inline pkg/kubelet/certificate/certificate_manager_test.go]: Do you prefer that to the deeper nesting I have here?

**Comment by rmmh** [comment]
Marking as 1.7 because this fixes test code.

**Pull request merged**